AndroidSDK/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
 <resources>
 
     <string name="app_name">Android SDK</string>
+    <string name="sdk_version">4.1.1</string>
 
-</resources>
+</resources>
```

Comment:
The sdk_version part look good but the rest of the file should be unchanged.

Reply:
I think this diff is showing like that because the XML header was being deleted, Ive fixed it so it should be the same now